reactapplication/src/card.jsx

```diff
@@ -10,11 +15,11 @@ export default function Card({ testimonial, cardIndex, index }) {
         <img className="CardImage" {...testimonial.image} />
         <p className="reviewer">{testimonial.name}</p>
         <div className="">
-          <img className="ratingImage" src="./images/star.png" alt="star" />
-          <img className="ratingImage" src="./images/star.png" alt="star" />
-          <img className="ratingImage" src="./images/star.png" alt="star" />
-          <img className="ratingImage" src="./images/star.png" alt="star" />
-          <img className="ratingImage" src="./images/star.png" alt="star" />
+          <div className="rating">
+            {[...Array(testimonial.rating)].map((e, i) => {
```

Comment:
`[...Array(testimonial.rating)]` means -> create an empty array of size 5 then create a shallow copy of that array.
Just `Array(testimonial.rating)` garda huncha